lib/utils/compose_ts.ts

```diff
@@ -918,7 +1047,7 @@ export async function checkBuildSecretsRequirements(
 	docker: Dockerode,
 	sourceDir: string,
 ) {
-	const [metaObj, metaFilename] = await loadBuildMetatada(sourceDir);
```

Comment:
😁

Reply:
This was driving my spellcheck plugin crazy :laughing: